dev/archetypes/maven-java/src/main/resources/archetype-resources/__service1Name__-impl/src/bundle/bundle.conf

```diff
@@ -0,0 +1,28 @@
+version = "1"
+name = "${service1Name}"
+compatibilityVersion = "1"
+system = "${service1Name}"
+systemVersion = "1"
+nrOfCpus = 0.1
```

Comment:
Yeah I know.  This is the default that sbt-conductr generates, I assume the ConductR team knew what they were doing when they wrote that, so I kept it as is.